tests/config_test.py

```diff
@@ -37,8 +8,14 @@
 import unittest
 import uuid
 
-from llbase import config
 from llbase import llsd
```

Comment:
why not import directly from llsd?  Or is this effectively to test llbase.llsd import capability?